### PR TITLE
(#3723) - add a "who's using" page

### PR DESCRIPTION
**Pull request by nolanlawson**
I think this is one of the first questions people
ask themselves when they're evaluating a piece of
software, so it's important for us to market ourselves.

The images were scaled down a bit to fit well on the page.
Feel free to tinker if you think it needs it @nickcolley.

Also we should definitely tweet and ask people to submit their
projects. I already emailed Chilean fishing boat folks to see
if we could get a statement from them. :)

### CODE REVIEW

File: docs/_includes/img.html [hunk 1]
```diff
@@ -4,6 +4,12 @@
   {% capture href %}{{ site.baseurl }}/static/img/{{ include.src }}{% endcapture %}
 {% endif %}
 
+{% if include.width %}
 <a href="{{ href }}" target="_blank">
-<img class="img-responsive" src="{{ site.baseurl }}/static/img/{{ include.src }}" alt="{{ include.alt }}">
+  <img class="img-responsive" width="{{include.width}}" src="{{ site.baseurl }}/static/img/{{ include.src }}" alt="{{ include.alt }}">
```
Comment (nickcolley) [inline docs/_includes/img.html]:
```html
<img
    {% if include.width %}
        width="{{include.width}}"
    {% endif %}
    class="img-responsive"
    src="{{ site.baseurl }}/static/img/{{ include.src }}"
    alt="{{ include.alt }}"
>
```
Does that not work?

Comment (nolanlawson) [inline docs/_includes/img.html]: it would, thanks!

**Comment by nickcolley** [comment]
Out of curiosity do Squarespace have a write up where they say they use PouchDB?

File: docs/_includes/img.html (hunk 1 above)
Comment (nolanlawson) [inline docs/_includes/img.html]: okay, had to make it a single line in order for it to work, but it does work

**Comment by nolanlawson** [comment]
Yup, they're listed here: https://software-licenses.squarespace.com/squarespace-blog-for-android

**Comment by nickcolley** [comment]
Let me check the width hardcoding btw, might be able to keep it scaling as well as restricting the size.

File: docs/users.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+{% include img.html width=200 src="cloudwall.png" alt="Cloudwall" %}
+
+[Cloudwall](http://cloudwall.me/) is an operating system for noBackend webapps, based on CouchDB and PouchDB.
+
+## MBTA Alerts
+
+{% include img.html width=100 src="mbta_alerts.jpeg" alt="MBTA Alerts" %}
+
+[MBTA Alerts](https://twitter.com/MBTA_Alerts) is a Twitter bot that thousands of Bostonians depend upon to be notified of delays in the MBTA transit system. Uses Node.js, PouchDB, and CouchDB.
```
Comment (calvinmetcalf) [inline docs/users.md]: tens of thousands, just saying

Comment (nolanlawson) [inline docs/users.md]: :smiley_cat:

**Comment by nickcolley** [comment]
Had a proper look and it looks good, tried to merge it but my usual method ```curl https://github.com/pouchdb/pouchdb/pull/3723.patch | git am -``` failed, any ideas?

**Comment by nolanlawson** [comment]
Maybe there's a conflict. I'll try to merge myself.

**Comment by nolanlawson** [comment]
09c32ba8369f666daf2bb2553605e55f2d610b33

**Comment by nolanlawson** [comment]
BTW my workflow is

```
git checkout branch-name
git rebase origin/master
git checkout master
git merge -
git push origin master
```

More steps, but I like to see what's happening at each step. :stuck_out_tongue:

**Comment by nickcolley** [comment]
That's my usual workflow to be honest, good to know though it's sane ha.

**Pull request closed without merging**